fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonHlProjectUtils.xtend

```diff
@@ -1281,6 +1281,7 @@ from PyTango import DevState
 «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»
         «protectedAreaHL(cls, "test_additional_imports")»
 «ENDIF»
+«cls.inheritedAdditionalImportHL»
```

Comment:
I don't understand why this is being added.  If I understand correctly, this will result in something like `from BaseDevice import BaseDevice`.   In the generated tests, the `BaseDevice` class isn't used, so it is unnecessary.